### PR TITLE
aligned loader center while loading pages

### PR DESCRIPTION
**Pull request by saikiranpatil**
### What
Created Loader Component and added it in all the pages

### Screenshot
252311966-c8f09628-d519-4e56-a00c-8f2f9b3a327b

### Fixes Issue
#668

### CODE REVIEW

**Comment by VaiTon** [comment]
Supersedes #669

File: src/App.jsx [hunk 1]
```diff
@@ -45,6 +44,7 @@ const LogoQuestionValidator = React.lazy(() =>
   import("./pages/logosValidator/LogoQuestionValidator")
 );
 const DashBoard = React.lazy(() => import("./pages/logosValidator/DashBoard"));
+const Loader = React.lazy(() => import("./pages/loader"));
```
Comment (VaiTon) [inline src/App.jsx:47]: I would import this statically, as it should be only a few Bytes and using a dynamic import for a fallback seems counterintuitive.

File: src/pages/loader/index.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import * as React from "react";
+import { Stack, CircularProgress, CssBaseline } from "@mui/material";
+
+const Loader = () => {
```
Comment (VaiTon) [inline src/pages/loader/index.jsx:4]:
```suggestion
export default function Loader() {
```

File: src/pages/loader/index.jsx [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+        justifyContent="center"
+        alignItems="center"
+      >
+        <CircularProgress />
+      </Stack>
+    </React.Fragment>
+  );
+};
+
+export default Loader;
```
Comment (VaiTon) [inline src/pages/loader/index.jsx:19]: Relative to the comment above, remove this

**Review by VaiTon — COMMENTED** [review]
Some minor issues, but otherwise good work :tada:

File: src/App.jsx (hunk 1 above)
Comment (Sudhanva-Nadiger) [reply src/App.jsx:47]: @VaiTon there is index.jsx page in pages folder which is importing all the pages ! I think we don't need that file anymore as we are importing everything directly in the app.jsx file.

Comment (saikiranpatil) [reply src/App.jsx:47]: i had imported everywhere statically, this ones left, i will change that

Comment (Sudhanva-Nadiger) [reply src/App.jsx:47]: I mean this file : https://github.com/openfoodfacts/hunger-games/blob/master/src/pages/index.jsx

Comment (Sudhanva-Nadiger) [reply src/App.jsx:47]: Okay ! Cool ! Shall I open a issue ?

Comment (VaiTon) [reply src/App.jsx:47]: You could even directly open a PR

Comment (Sudhanva-Nadiger) [reply src/App.jsx]: Umm cool !

Comment (Sudhanva-Nadiger) [reply src/App.jsx]: @VaiTon the pr is ready :)

**Pull request merged**